openstack/compute/v2/servers/requests.go

```diff
@@ -125,9 +125,10 @@ type CreateOpts struct {
 	// Metadata [optional] contains key-value pairs (up to 255 bytes each) to attach to the server.
 	Metadata map[string]string
 
-	// Personality [optional] includes the path and contents of a file to inject into the server at launch.
-	// The maximum size of the file is 255 bytes (decoded).
-	Personality []byte
+	// Personality [optional] includes a list of maps with the path and contents
+	// of a file to inject into the server at launch. Contents should be
+	// base64 encoded. The maximum size of the file is 255 bytes (decoded).
+	Personality []map[string]string
```

Comment:
Since it's fixed to be only `path` and `contents`, maybe we can have a `Personality` struct:
```go
type Personality struct {
    Path string
    Contents string
}
```

and then have the field definition in the `CreateOpts` (and the like) be
``` Personality []Personality```